.github/workflows/scripts/geth-commit.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# This script finds the base commit corresponding to a geth tag
+# from which libevm was last based on, and returns it to the caller.
+# If an error occurs, this one is echo-ed and the script exits with code 1.
+
+git remote add geth https://github.com/ethereum/go-ethereum.git
+git fetch geth 'refs/tags/*:refs/tags/*'
+
+geth_tags=$(git ls-remote --tags --sort=-version:refname geth "v[1-9]*" | awk '{print $2}' | sed 's/refs\/tags\///')
```

Comment:
```suggestion
geth_tags=$(git ls-remote --tags --sort=-version:refname geth "v[1-9]*" | awk '{print $2}' | awk -F'/' '{print $NF}'
```

Doesn't require knowledge of the output to understand the modification.